instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+    1.  the date and moderator of the discussion session they have taken part in
+        *   we currently store both values in one column
+	*   we do not record both a Software Carpentry and a Data Carpentry discussion session (though we should)
+    1.  the date on which they passed their Software Carpentry lesson demo (if any)
+	*   this date becomes the award date of the PDF certificate
+        *   we do not currently record the name of the person who was their instructor, or the name of the person who did their final test
+    1.  the date on which they passed their Data Carpentry lesson demo (if any)
+        *   the same caveats apply
+1.  Mentors offer lesson discussion sessions,
+    and trainees sign up for them,
+    on an Etherpad.
+1.  Trainers offer checkout sessions on a second Etherpad.
+    Trainees sign up for sessions on that Etherpad,
+    which we also presently use for taking notes during their sessions.
+1.  Discussion and checkout sessions are marked in Software Carpentry's community calendar
+    (a Google calendar).
+
+## Issues
+
+1.  As of Jan 2016,
+    the form for individuals to apply to take part in training has been disabled
+    while we try to clear our backlog (and come up with a better process).
+1.  We used a Google web form and a spreadsheet for group applications and sorted manually.
+    We are not systematically tracking whether groups run their promised workshops or not.
+1.  Negotiations with partners about instructor training sessions are tracked in a Google Docs spreadsheet.
+    Schedules are drawn up manually.
+1.  We have never had a way to help individual training applicants find one another to form groups.
+1.  We don't have any way to determine whether individual applicants are associated with our partners.
+    (The applicants may not know that their organization is a partner.)
+1.  Trainees are supposed to email the head of training when they submit their lesson change.
+    The head of training then updates the spreadsheet
+    and mails the trainee instructions on how to sign up for a discussion session on the lesson discussion Etherpad.
+1.  The head of training recruit discussion leaders by mailing the mentoring list and a few individuals
+    to ask for volunteers.
+    When trainees want a discussion session,
+    and there isn't one scheduled for their chosen lesson(s),
+    they put themselves at the top of the discussion Etherpad.
+    The head of training then asks mentors to set up more discussion sessions
+    and gives the mentors the email addresses of people who want one of that kind.
+1.  People leading discussion sessions report back to head of training by email,
+    who then updates the trainee's spreadsheet entry
+    and sends them mail telling them to sign up to do a demo on the lesson demo Etherpad.
+1.  Using one Etherpad both to manage sign-up for demo sessions
+    and to take notes during those sessions
+    is confusing.
```

Comment:
Using Etherpad for registration is clumsy, so I'd like to get rid of it:

1.  Doesn't notify trainees when new sessions of the kind they want appear.
2.  Doesn't notify discussion leaders when trainees want sessions of the 
kind they're able to offer.
3.  Doesn't notify us (admins) when sessions have taken place, who got 
through, etc.

Parsing the text of the Etherpad every once in a while isn't a solution: 
hand-written will inevitably fail to conform to style rules, 
name-matching is hard, etc.

Separate Etherpads to take notes during sessions (one per discussion 
leader, recycled) would be fine.